pyigm/cgm/models.py

```diff
@@ -0,0 +1,240 @@
+"""  Module for CGM models
+"""
+
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+import pdb
+
+from astropy import units as u
+from astropy import constants as const
+
+try:
+    basestring
+except NameError:  # For Python 3
+    basestring = str
+
+def rad3d2(xyz):
+    return xyz[0]**2 + xyz[1]**2 + xyz[-1]**2
```

Comment:
Radius calculation would be misleading if xyz origin is not defined as (0, 0, 0).